tests/Aquifer.Public.API.UnitTests/ResourcesCollectionsGetEndpointValidatorTests.cs

```diff
@@ -0,0 +1,58 @@
+using Aquifer.Public.API.Endpoints.Resources.Collections.Get;
+using FluentValidation.TestHelper;
+
+namespace Aquifer.Public.API.UnitTests;
+
+public sealed class ResourcesCollectionsGetEndpointValidatorTests
```

Comment:
I tried out the `FluentValidation.TestHelper` with XUnit and am a fan.  It was pretty quick to throw these tests together and they are all tests I would have had to perform manually every time that I changed the validator logic.

I went away from "best practices" and just have two methods, one for bad requests and one for good requests.  This can lend itself to tests that have multiple properties causing a failure but I think it's (a) easier to read because there is much less test code and (b) the pitfalls can be mitigated by only changing one property to invalid at a time.